libs/api-client-bear/src/project.ts

```diff
@@ -1,6 +1,21 @@
 // (C) 2007-2022 GoodData Corporation
 import { invariant } from "ts-invariant";
-import { GdcProject, GdcUser, GdcUserGroup, GdcAccessControl } from "@gooddata/api-model-bear";
+import {
+    IProjectId,
+    IUserProjectsResponse,
+    IUserProjectsParams,
+    ITimezone as IBearTimezone,
```

Comment:
Why is this rename needed? Looks suspicious :)

Reply:
Type from api-client-bear ITimeZone is different than ITimezone from from interfaces this rename is needed.